src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div  class="stars stars--0">
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star "></div>
+      <div class="stars__star"></div>
+    </div>
+    <div  class="stars stars--1">
+      <div class="stars__star modifier"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
redundant modifier
remove for all elements
for amount of active stars will be responsible stars--1 ... stars--5 modifiers and `:nth-child` pseudo class